internal/legacy/dockerregistry/inventory.go

```diff
@@ -1752,6 +1752,7 @@ func (sc *SyncContext) Promote(
 
 					opts := []crane.Option{
 						crane.WithAuthFromKeychain(gcrane.Keychain),
+						crane.WithUserAgent("k8s-image-promoter"),
```

Comment:
Can we const this away in types/image?

```suggestion
						crane.WithUserAgent(image.UserAgent),
```

Reply:
And can we name it `kpromo`  :pray:

Reply:
done, thanks for the feedback